youths/schema.py

```diff
@@ -57,11 +57,17 @@ class YouthProfileType(DjangoObjectType):
     membership_status = MembershipStatus(
         description="Membership status based on expiration and approved_time fields"
     )
+    renewable = graphene.Boolean(
+        description="Tells if the membership is currently renewable or not"
+    )
 
     class Meta:
         model = YouthProfile
         exclude = ("id", "approval_token", "language_at_home")
 
+    def resolve_renewable(self, info, **kwargs):
+        return self.expiration != calculate_expiration(date.today())
```

Comment:
`!=`?

Reply:
ah, I guess not. I think I get it now.

Reply:
Yeah it's a bit tricky. Maybe some kind of  `is_renewable` helper would have been nice but as it's a one liner I guess that'll do.